Polish/przygoda/jedzenie/wiedzmin/placki/placki.md

```diff
@@ -1,2 +1,7 @@
 Nie pytajac Cie o zdanie, nerd z nadludzka sila ciagnie Cie w strone ciemnego pomieszczenia.
 Nie masz wiele czasu, a opcji niewiele. Co robisz??
+
+najlepszą obroną jest atak - [rzucasz zaklecie IGNI](igni/igni.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.